docs/contribution_guide/contributing_example_notebooks.rst

```diff
@@ -0,0 +1,16 @@
+Contributing Example Notebooks
+------------------------------
+
+To contribute new example notebooks please submit a pull request. We have
+certain requirements that need to be satisfied to contribute a notebook. Those
+include:
+
+* clear structure
+* appropriate framing of the problem while respecting that fairness is a
+  fundamentally sociotechnical challenge
+* usage of at least some part of Fairlearn
```

Comment:
Should we capitalize the first letter for consistency with other lists and put period at the end? Since these are not sentences, I'm a bit ambivalent.